test/e2e/e2e_suite_test.go

```diff
@@ -77,6 +78,9 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		if kubeconfigPath == "" {
 			kubeconfigPath = os.Getenv("HOME") + "/.kube/config"
 		}
+		k8sContextName := e2eConfig.GetVariable("KUBECONFIG_DEFAULT_CONTEXT_NAME")
+		err := SwitchKubeconfigContext(kubeconfigPath, k8sContextName)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Yeah that makes senses. It would make the errors a bit more difficult to spot if user forgets to set the kubeconfig into correct context beforehands, but perhaps that's a bit of an odd case.